Config/BuildSettings.swift

```diff
@@ -280,6 +280,13 @@ final class BuildSettings: NSObject {
     static let roomScreenAllowMediaLibraryAction: Bool = true
     static let roomScreenAllowStickerAction: Bool = true
     static let roomScreenAllowFilesAction: Bool = true
+    static var roomScreenAllowPollsAction: Bool {
+        guard #available(iOS 14, *) else {
+            return false
+        }
+        
+        return true
```

Comment:
Yes, I would like to make it false before merging this PR. I don't think we should wait until the rest is implemented, I want to avoid conflicts if possible.